.github/workflows/manifest.yml

```diff
@@ -29,5 +29,14 @@ jobs:
           checkout-path: 'ncs/nrf'
           label-prefix: 'manifest-'
           verbosity-level: '1'
-          labels: 'manifest'
+          labels: >
+            manifest,
+            CI-all-test:zephyr;nrfxlib,
```

Comment:
Could you add some more comments here on how to use.
I can deduce by myself, but I think it's nice to have a bit more description for other to read.

Also maybe update the commit message with info on which relationship this commit defines.

Reply:
Thanks @tejlmand. I added a comment with examples and instructions. I'm not sure what you are asking for the commit message, but I added a bit of rationale.